README.md

```diff
@@ -163,10 +170,12 @@ initialization scripts (and later daemons) to pick from.
 ovs-vsctl set Open_vSwitch . external_ids:k8s-api-server="$OVN_CENTRAL_IP:8080"
 ```
 
-Install the following additional packages for python.
+Clone this repository and install the executables.
```

Comment:
Technically you're not installing the executables, but the package.
The executables are just simple python modules that load the actual "script" module and execute it:

#!/usr/bin/python
# EASY-INSTALL-DEV-SCRIPT: 'ovn-kubernetes==0.0.1.dev26','ovn-k8s-overlay'
__requires__ = 'ovn-kubernetes==0.0.1.dev26'
__import__('pkg_resources').require('ovn-kubernetes==0.0.1.dev26')
__file__ = '/home/salvatore/ovn-kubernetes/bin/ovn-k8s-overlay'
exec(compile(open(__file__).read(), __file__, 'exec'))

Personally I prefer using console-scripts which result in something nicer (but this is a story for another PR): 
#!/usr/bin/python
# PBR Generated from u'console_scripts'

import sys
from ovn_k8s.bin.ovn_k8s_watcher import main
if __name__ == "__main__":
    sys.exit(main())